.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```

Comment:
I feel like it would be useful to already have the env file so everydidn't have to create it..

Reply:
No. The .env holds confidential information so we can't have it on main. This kinda stuff will have to be shared thru PMs. @johan253 Can you make a .sample.env just with empty fields so people can create their local .env-s. It will be the same as your local .env but without the info.

Reply:
Typically the .env file of a project includes sensitive secrets, such as API Keys, passwords, and private URL's. We do not want this information publicly accessible on GitHub.

Reply:
I'll add the env vars to amplify tomo so it woks on there. I wonder if there is a way to run an int test to see if the server will run on AWS ://

Reply:
I see, gotcha